NET_project/Benchmarks/Arcfour.cs

```diff
@@ -1,43 +1,47 @@
 ﻿using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
 
 namespace NET_project.Benchmarks;
 
+[SkipLocalsInit]
 public unsafe struct ArcfourNET : IJob
 {
 	public uint iterations;
 	public int result;
 
-	public void Run()
-	{
+	public void Run() {
 		result = Arcfour(iterations);
 	}
 
-	private int Arcfour(uint iterations)
-	{
+	private static int Arcfour(uint iterations) {
 		const int keyLength = 5;
 		const int streamLength = 10;
 
-		byte[] state = new byte[256];
-		byte[] buffer = new byte[64];
-		Span<byte> key = stackalloc byte[keyLength];
-		Span<byte> stream = stackalloc byte[streamLength];
-
-		key[0] = 0xDB;
-		key[1] = 0xB7;
-		key[2] = 0x60;
-		key[3] = 0xD4;
-		key[4] = 0x56;
-
-		stream[0] = 0xEB;
-		stream[1] = 0x9F;
-		stream[2] = 0x77;
-		stream[3] = 0x81;
-		stream[4] = 0xB7;
-		stream[5] = 0x34;
-		stream[6] = 0xCA;
-		stream[7] = 0x72;
-		stream[8] = 0xA7;
-		stream[9] = 0x19;
+		// Using span here would be more complex due to the general
+		// inability to prove that several accesses are safe due to
+		// the index being the value of another buffer i.e. x[y[n]]
+
+		byte* state = (byte*)NativeMemory.AlignedAlloc(256, 8);
+		byte* buffer = (byte*)NativeMemory.AlignedAlloc(64, 8);
+		byte* key = stackalloc byte[keyLength] {
+			0xDB,
+			0xB7,
+			0x60,
+			0xD4,
+			0x56,
+		};
+		byte* stream = stackalloc byte[streamLength] {
```

Comment:
Notably this is unused in both .NET and C. It's not clear if that's a bug in the benchmark or not.